int/api/websites/all.go

```diff
@@ -78,6 +79,9 @@ func Registry(config config.AppConfig) ([]*models.Registry, error) {
 		websiteStorerAddress := websiteValue[indexOfWebsiteAddress]
 		websiteDescription := websiteValue[indexOfWebsiteDescription]
 
+		// Prevent XSS by escaping special characters in websiteDescription
+		escapedDescription := template.HTMLEscapeString(websiteDescription)
+
```

Comment:
can you factorize this code  in a dedicated function? and use it in the second place its needed

Reply:
https://github.com/massalabs/station/issues/792